app/views/estimate_flow/level_of_help.html.slim

```diff
@@ -18,6 +18,8 @@
     p.govuk-body = t(".also_includes")
     ul.govuk-list.govuk-list--bullet
       - t(".controlled_chambers").each do |bullet|
+        - next if bullet.downcase.include?("immigration") && !FeatureFlags.enabled?(:asylum_and_immigration)
```

Comment:
I'm not sure if there is a better place for this logic rather than the view - but I couldn't see any view_object files to use instead

Reply:
refactored based on my own comment :)

Reply:
And it's a good refactor! We have been a bit lax about letting quite a bit of logic sneak into the view files. The `form` classes are a better place, but we don't have one of them for every page. Maybe in the next dev team time we could chat about better approaches? I've not used view_objects, but from the name they sound like they might be just the ticket. Perhaps you could walk us through how you've used them before?

Reply:
I've just switched the implementation back to having the conditional with the feature flag in the view - chatted with @willc-work and we decided that is probably easier for the time being. But definitely up for chatting about how we can move the logic elsewhere/into a view object - they're quite nice if you know you're going to need to manipulate the UI a bit more